libs/plugin-cache/src/lib/create-cache-plugin.spec.ts

```diff
@@ -187,6 +182,31 @@ describe('CachePlugin', () => {
     })
   );
 
+  it('should not set cache entry when storage outsized', async () => {
```

Comment:
Actually, I think that the cache entry should be added and older entries or Least Recently Used entries should be removed.

Reply:
This should be handled in the storage because for instance, if we use redis as a storage, it already implements LRU or Least Frequenty Used : https://redis.io/topics/lru-cache